src/Cake.Frosting.Template/templates/cakefrosting/.template.config/template.json

```diff
@@ -0,0 +1,11 @@
+{
+  "author": "Patrik Svensson (patrik@patriksvensson.se)",
```

Comment:
Do we want to keep @patriksvensson  as author here or switch to same as for the other runners?

Reply:
Switch to the same as other runners.

Reply:
This is in the template manifest not nuget packages. But can investigate charging this.

Reply:
Changed to the same as NuGet packages.
Doesn't look like there're any special rules around this property, so probably just works.
![image](https://user-images.githubusercontent.com/1647294/96083869-535b4e80-0ebe-11eb-9011-b1b45aa6c8e2.png)